### DISCUSSION TITLE
Should ChatGPT be added?

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#362] [Ideas]
Is it worth adding ChatGPT and what benefits do you see in adding ChatGPT to the application?

**Comment by Faq** [comment]
1. I guess as implementation process for you it could be interesting if you willing to waste time on it. As it will take extra time for you to maintain and fix things, my guess is it is more up to you. 

2. Is it worth for this project? I don't think so as this is not IDE, but more like note/snippet tool. For now I don't see what would be the gain of such AI on this project.

**Comment by LukasVolgger** [comment]
It probably depends on what the AI ​​is used for.  I think the implementation would definitely make sense for automatically generated descriptions and tags, as this takes a lot of time to maintain manually.
